install.sh

```diff
@@ -45,23 +103,110 @@ cd ~/wordlists/ && wget https://raw.githubusercontent.com/Bo0oM/fuzz.txt/master/
-go get -u github.com/tomnomnom/assetfinder 2> /dev/null
-GO111MODULE=on go get -u -v github.com/bp0lr/gauplus 2> /dev/null
-GO111MODULE=on go get -v github.com/projectdiscovery/httpx/cmd/httpx 2> /dev/null
-go get -v github.com/OWASP/Amass/v3/... 2> /dev/null
-go get github.com/tomnomnom/waybackurls 2> /dev/null
-go get github.com/Emoe/kxss 2> /dev/null
-go get github.com/haccer/subjack 2> /dev/null
-go get -u github.com/tomnomnom/qsreplace 2> /dev/null
-cd ~/tools/temp/ && git clone https://github.com/projectdiscovery/dnsx.git; cd dnsx/cmd/dnsx; go build; mv dnsx /usr/bin/ &> /dev/null 
-cd ~/tools/temp/ && git clone https://github.com/hahwul/dalfox && cd dalfox && go install 2> /dev/null
-cd ~/tools/temp/ && git clone https://github.com/dwisiswant0/crlfuzz && cd crlfuzz/cmd/crlfuzz && go build . && sudo mv crlfuzz /usr/bin/ 2> /dev/null
-cd ~/tools/temp/ && git clone https://github.com/projectdiscovery/nuclei.git && cd nuclei/v2/cmd/nuclei && go build && mv nuclei /usr/bin/ 2> /dev/null
-cd ~/tools/temp/ && git clone https://github.com/projectdiscovery/subfinder.git && cd subfinder/v2/cmd/subfinder && go build && mv subfinder /usr/bin/ 2> /dev/null
-cd ~/tools/temp/ && wget https://github.com/OWASP/Amass/releases/download/v3.13.4/amass_linux_i386.zip && unzip amass_linux_i386.zip && cd amass_linux_i386 && mv amass /usr/bin/ 2> /dev/null
+if ! command -v anew &> /dev/null
+then
+    echo "anew could not be found"
+    go get -u github.com/tomnomnom/anew@latest 2> /dev/null
+fi
+
+if ! command -v gf &> /dev/null
+then
+    echo "gf could not be found"
+    go get -u github.com/tomnomnom/gf@latest 2> /dev/null
+fi
+
+if ! command -v assetfinder &> /dev/null
+then
+    echo "assetfinder could not be found"
+    go install github.com/tomnomnom/assetfinder@latest 2> /dev/null
+fi
+
+if ! command -v aquatone &> /dev/null
+then
+    echo "aquatone could not be found"
+    go install github.com/michenriksen/aquatone@latest 2> /dev/null
+fi
+
+if ! command -v httpx &> /dev/null
+then
+    echo "httpx could not be found"
+    go install -u github.com/projectdiscovery/httpx/cmd/httpx@latest 2> /dev/null
+fi
+
+if ! command -v kxss &> /dev/null
+then
+    echo "kxss could not be found"
+    go install github.com/Emoe/kxss@latest 2> /dev/null
+fi
+
+if ! command -v subjack &> /dev/null
+then
+    echo "subjack could not be found"
+    go install github.com/haccer/subjack@latest 2> /dev/null
+fi
+
+if ! command -v qsreplace &> /dev/null
+then
+    echo "qsreplace could not be found"
+    go install -u github.com/tomnomnom/qsreplace@latest 2> /dev/null
+fi
+
+if ! command -v waybackurls &> /dev/null
+then
+    echo "waybackurls could not be found"
+    go install github.com/tomnomnom/waybackurls@latest 2> /dev/null
+fi
+
+if ! command -v gauplus &> /dev/null
+then
+    echo "gauplus could not be found"
+    GO111MODULE=on go get -u -v github.com/bp0lr/gauplus 2> /dev/null
+fi
+
+if ! command -v httpx &> /dev/null
```

Comment:
There's a duplicate check for `httpx` on line 130. Please remove one of them to avoid redundancy.